drivers/hmis/app/graphql/types/hmis_schema/enums/hud.rb

```diff
@@ -470,12 +470,6 @@ class VamcStationNumber < Types::BaseEnum
     hud_enum HudUtility2024.vamc_station_numbers
   end
 
-  class NoPointsYes < Types::BaseEnum
-    description 'V7.1'
-    graphql_name 'NoPointsYes'
-    hud_enum HudUtility2024.no_points_yes_options
-  end
```

Comment:
This seems to be missing.

Reply:
thats ok it is not used in the graphql schema. IDK what this was but question V7.1 uses the standard 1.7 list.